### ISSUE TITLE
CheckingResponse proto does not match API response

### ISSUE THREAD

**Issue by Tobiaqs** [Chavithra/degiro-connector#10]
![Screenshot from 2021-03-29 09-39-58](https://user-images.githubusercontent.com/2378369/112802701-c6264380-9072-11eb-872e-7dfbddcd8d86.png)
This is what the `checkOrder` endpoint returns for me. If I try to create an order I get the error that transactionAutoFxSurcharges is not part of the CheckingResponse proto. I fixed this for myself by matching the proto to the API response.
I have a custody account.

**Comment by Chavithra** [comment]
Hi,

thanks for reporting that bug.

I was able to reproduce this issue.

The format of the `CheckingResponse` returned by `Degiro's API` seems to depend on the product.

I have updated the library in order to : 
- make the `CheckingResponse`  less restrictive
- parse both formats properly

By the way there is a `raw` parameter in most of the methods in order to get the raw API response: 
```python
checking_response = trading_api.check_order(order=order, raw=True)
```
